packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -49,6 +50,9 @@ const TransitionGroupImpl: ComponentOptions = {
     let prevChildren: VNode[]
     let children: VNode[]
 
+    const controller = new AbortController()
```

Comment:
`AbortController` is supported in Chrome > 66, which will break current compatibility. We can only use `el.removeEventListener`.

Reply:
So, can we wrap `addEventListener` in a second function that returns a `removeEventListener`, and then save that function to call it when the component is unmounted?

```ts
export function addEventListener(
  el: Element,
  event: string,
  handler: EventListener,
  options?: EventListenerOptions
) {
  el.addEventListener(event, handler, options)

  return () => el.removeEventListener(event, handler, options)
}

```

Reply:
Thank you for your reply, I have edited it